docs/1-Using-Fleet/3-REST-API.md

```diff
@@ -630,6 +637,188 @@ Gets the current SSO configuration.
+        "code_name": "",
+        "uptime": 11202000000000,
+        "memory": 2085326848,
+        "cpu_type": "6",
+        "cpu_subtype": "142",
+        "cpu_brand": "Intel(R) Core(TM) i5-8279U CPU @ 2.40GHz",
+        "cpu_physical_cores": 4,
+        "cpu_logical_cores": 4,
+        "hardware_vendor": "",
+        "hardware_model": "",
+        "hardware_version": "",
+        "hardware_serial": "",
+        "computer_name": "259404d30eb6",
+        "primary_ip": "172.19.0.4",
+        "primary_mac": "02:42:ac:13:00:04",
+        "distributed_interval": 10,
+        "config_tls_refresh": 10,
+        "logger_tls_period": 10,
+        "additional": {},
+        "enroll_secret_name": "bar",
+        "status": "offline",
+        "display_text": "259404d30eb6"
+    }
+}
+```
+
+### Get host by identifier
+
+Returns the information of the host specified using the `uuid`, `osquery_host_id`, `hostname`, or
+`node_key` as an identifier
+
+`GET /api/v1/kolide/hosts/identifier/{identifier}`
+
+#### Parameters
+
+| Name                  | Type   | In   | Description                                                     |
+| --------------------- | ------ | ---- | --------------------------------------------------------------- |
+| identifier                 | integer or string | path | **Required**. The host's `uuid`, `osquery_host_id`, `hostname`, or `node_key`|
+
+#### Example
+
+`GET /api/v1/kolide/hosts/identifier/f01c4390-0000-0000-a1e5-14346a5724dc`
+
+
+##### Default response
+
+`Status: 200`
+
+```
+{
+    "host": {
+        "created_at": "2021-01-19T18:04:12Z",
+        "updated_at": "2021-01-19T20:21:27Z",
+        "id": 121,
+        "detail_updated_at": "2021-01-19T20:04:22Z",
+        "label_updated_at": "2021-01-19T20:04:22Z",
+        "last_enrolled_at": "2021-01-19T18:04:12Z",
+        "seen_time": "2021-01-19T20:21:27Z",
+        "hostname": "259404d30eb6",
+        "uuid": "f01c4390-0000-0000-a1e5-14346a5724dc",
+        "platform": "ubuntu",
+        "osquery_version": "2.10.2",
+        "os_version": "Ubuntu 14.4.0",
+        "build": "",
+        "platform_like": "debian",
+        "code_name": "",
+        "uptime": 11202000000000,
+        "memory": 2085326848,
+        "cpu_type": "6",
+        "cpu_subtype": "142",
+        "cpu_brand": "Intel(R) Core(TM) i5-8279U CPU @ 2.40GHz",
+        "cpu_physical_cores": 4,
+        "cpu_logical_cores": 4,
+        "hardware_vendor": "",
+        "hardware_model": "",
+        "hardware_version": "",
+        "hardware_serial": "",
+        "computer_name": "259404d30eb6",
+        "primary_ip": "172.19.0.4",
+        "primary_mac": "02:42:ac:13:00:04",
+        "distributed_interval": 10,
+        "config_tls_refresh": 10,
+        "logger_tls_period": 10,
+        "additional": {},
+        "enroll_secret_name": "bar",
+        "status": "offline",
+        "display_text": "259404d30eb6"
+    }
+}
+```
+
+### Delete host
+
+Deletes the specified host from Fleet.
```

Comment:
Add something like
```
Note that a deleted host will fail authentication with the previous node key, and in most osquery configurations will attempt to re-enroll automatically. If the host still has a valid enroll secret, it will re-enroll successfully.
```